### DISCUSSION TITLE
Bash code block in the documentation redirect to top

### DISCUSSION THREAD

**Discussion by IDevelopAPI** [fastapi/fastapi#12420] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
<a href="#" data-terminal-control="">fast →</a>
```

### Description

[URL](https://fastapi.tiangolo.com/tutorial/sql-databases/#install-sqlmodel)

👉 When I click "fast" or "restart" - I'm redirected to the top of the page. 
❓ Maybe it is because of the anchors "#" in the href. 
❎ It's inconvenient when u read, e.g., near the center of the page.

Any ideas or suggestions on how to fix this?

### Operating System

Windows

### Operating System Details

Google Chrome Version 129.0.6668.90 (Official Build) (64-bit)

### FastAPI Version

0.115.0

### Pydantic Version

2.9.2

### Python Version

3.9

**Comment by Reemyos** [comment]
In Safari and Brave browsers this doesn't happen. Maybe it has to do with your Chrome version. Try to upgrade if available and if not maybe downgrade if it's important enough for you.

**Reply by IDevelopAPI** [reply]
Thank you for tthe advise, but for me to use Chrome is important.
